frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -161,7 +190,7 @@ function mostRecentFirst(action: WritebackAction) {
 }
 
 const ACTION_MENU_TRIGGER_PROPS = {
-  "data-testid": "new-action-menu",
+  "data-testid": "actions-menu",
```

Comment:
What do you think about using `aria-label: "Actions menu"` instead? This would make this button more accessible too.